#!/usr/bin/python
# -*- coding: utf-8 -*-

L = {
	"READING_FILE": "Reading %s...",
	"READING_STRINGBLOCK": "Reading stringblock from %s...",
	"TOTAL_ROWS": "%i rows total",
	"USING_GENERATED_STRUCTURE": "Using generated structure for file %s, build %i",
	"USING_STRUCTURE": "Using %s structure build %i",
	"WRITTEN_BYTES": "Written %i bytes at %s",
	
	"BUILD_NOT_SET": "Build not set for a DBC file; assuming default structure.",
	"COLUMN_NOT_FOUND": "Column %s not found",
	"FILENAME_NOT_SPECIFIED": "wdbc.new requires a 'name' argument to function correctly",
	"FILETYPE_NOT_WRITABLE": "%s is not a writable filetype",
	"MULTIPLE_ROW_INSTANCE": "Multiple instances of row #%i found",
	"NO_STRUCTURE_FOUND": "No structure was found for file %r",
	"PATH_NOT_SET": "DBFile.path needs to be set before initiating parsing",
	"PATH_NOT_VALID": "DBFile.path needs to be a valid file before initiating parsing",
	"RECLEN_NOT_RESPECTED": "Reclen not respected for row %i. Expected %i, read %i. (%+i)",
	"UNREADABLE_FILE": "Unreadable file",
	"SUBSTRING_NOT_FOUND": "No string was found at address %i. This is most often caused by a bad structure. Doublecheck your StringFields!",
}
